templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/PageMenuItem.vue

```diff
@@ -95,18 +96,22 @@ export default {
         !node.unlocked
       )
     },
-    handleTitleClick() {
-      this.scrollToRow()
+    nestedItemClick() {
+      this.itemClick()

```

Comment:
Why have this function when you can just call `itemClick` (or `handleItemClick`) directly